app/Http/Controllers/Api/PermissionController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Api\UpdateRolePermissionsRequest;
+use App\Http\Requests\Api\UpdateUserRolesRequest;
+use App\Models\User;
+use Spatie\Permission\Models\Role;
+
+class PermissionController extends Controller
+{
+    // Update Role Permission
+    public function updateRolePermissions(UpdateRolePermissionsRequest $request)
```

Comment:
this is incorrect place to have this function. The functionality to update permissions for a role should be in the RoleController. `RoleController` is the place where any action related to user should be put in.